chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/state/LeftRoomReducer.kt

```diff
@@ -0,0 +1,15 @@
+package com.pusher.chatkit.rooms.state
+
+import com.pusher.chatkit.state.ChatkitState
+import com.pusher.chatkit.state.LeftRoom
+import org.reduxkotlin.reducerForActionType
+
+internal val leftRoomReducer =
+    reducerForActionType<ChatkitState, LeftRoom> { state, action ->
+        checkNotNull(state.joinedRoomsState)
+
+        state.with(JoinedRoomsState(
+                state.joinedRoomsState.rooms - action.roomId,
+                state.joinedRoomsState.unreadCounts - action.roomId
+        ), state.auxiliaryState.with(action))
```

Comment:
```suggestion
        state.with(
            JoinedRoomsState(
                state.joinedRoomsState.rooms - action.roomId,
                state.joinedRoomsState.unreadCounts - action.roomId
            ),
            state.auxiliaryState.with(action)
        )
```
using more vertical real estate but more legible I think, it's also closer to what the IDE reformats it into

Reply:
👍 have refactored the RoomDeletedReducer to match as they're the same reducer with a different name.